fleetspeak/src/client/https/streaming.go

```diff
@@ -339,6 +339,7 @@ func (c *StreamingCommunicator) connect(ctx context.Context, host string, maxLif
 		c.cctx.Stats().OutboundContactData(host, len(buf), err)
 		return fail(err)
 	}
+	log.Infof("POST to %v succeeded with status: %v and protocol: %v", host, resp.StatusCode, resp.Proto)
```

Comment:
If this log should be kept, we should make it use verbose level 2.

Reply:
Thanks a lot Torben, will update accordingly.